rules.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: Community Rules
+permalink: /rules/
+---
+
+##ITC should be considered SFW (Safe for Work). As a rule of thumb, before you post anything, consider if you'd want a co-worker or your boss to see it.
+
+###The following is not allowed:
+
+- Any content which is not in keeping with the Code of Conduct.
+- Blatantly sexual content (pornography, nudity, etc.)
+- Misogyny, ableism, sexism racism, nationalist insults etc.
+- Personal abuse.
+- Links to anything which would break the rules if posted directly.
+- Posting any private phone numbers.
+- Advocating criminality or violence.
+- Trolling (A 'troll' is someone who posts controversial, inflammatory or off-topic messages with the primary intent of provoking other users. These users should be reported to admins immediately.)
+- Posting libellous material.
+- Txt spk (text speak).
```

Comment:
Why do you think so?

What I find happens when you're not specific, based on my experience moderating another online community, is that you end up with "grey-areas" - These grey areas make it harder for admins/mods to do their jobs as it becomes less about what is and isn't allowed, and more about an admins discretion. This can then lead to some admins being seen as heavy handed, or could also lead to confusion among admins as to what is and is not ok.

I'm completely open to debate though.